pkgs/development/compilers/ghc/nokinds.nix

```diff
@@ -1,83 +0,0 @@
-{ stdenv, fetchgit, bootPkgs, perl, gmp, ncurses, libiconv, autoconf, automake, happy, alex }:
```

Comment:
If you remove the compiler, please remove the corresponding package set from `pkgs/top-level/haskell-packages.nix`, too.